scanners/reactnative/expo.go

```diff
@@ -244,86 +239,20 @@ func (scanner *Scanner) expoConfigs() (models.BitriseConfigMap, error) {
 			"DEVELOPMENT_TEAM = $BITRISE_IOS_DEVELOPMENT_TEAM"},
 	)
 
-	if !scanner.hasTest {
```

Comment:
Removed it was duplicate code. Now there always a deploy workflow, so no distinction needed.